### DISCUSSION TITLE
This is only for Arm64?

### DISCUSSION THREAD

**Discussion by RumblePen** [memstechtips/UnattendedWinstall#51] [Q&A] [answered]
I can't use this with an x64 processor?

**Comment by memstechtips** [comment, marked as answer]
You can, just click on the blue text [IoT-LTSC-Like](https://github.com/memstechtips/UnattendedWinstall/blob/main/IoT-LTSC-Like/autounattend.xml) to take you to the correct file. The normal blue text is x64 and the [arm64](https://github.com/memstechtips/UnattendedWinstall/blob/main/IoT-LTSC-Like/arm64/autounattend.xml) is for arm processors.
